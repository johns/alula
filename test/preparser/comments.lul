#:
This program was created for the purpose of adding x and y...
and then printing the result.
:#


function sum: num x, num y {
  return: x + y    # Adds two nums together and returns the result
}

print: sum: 4, 3
