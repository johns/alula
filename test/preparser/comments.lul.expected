function sum: num x, num y {
⇨return: x + y
⇦}

print: sum: 4, 3
